Library/Homebrew/formula_creator.rb

```diff
@@ -168,11 +168,9 @@ def install
         <% elsif mode == :go %>
             system "go", "build", *std_go_args(ldflags: "-s -w")
         <% elsif mode == :meson %>
-            mkdir "build" do
-              system "meson", *std_meson_args, ".."
-              system "ninja", "-v"
-              system "ninja", "install", "-v"
-            end
+            system "meson", "setup", "build", *std_meson_args
```

Comment:
There are a couple variations of `meson setup` command due to how positional arguments can be placed anywhere and `setup` could be omitted.

Now, `meson` has deprecated the default behavior:
```
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
```

I chose the most common order for `meson setup ...` command we use in homebrew-core repo.